scripts/traverse.ts

```diff
@@ -236,6 +249,13 @@ if (esMain(import.meta)) {
           type: 'boolean',
           default: process.stdout.isTTY,
         })
+        .option('deprecated', {
+          alias: 'dp',
```

Comment:
```suggestion
          alias: 'x',
```

Perhaps `-x` would be a bit more fitting?